### PR TITLE
Aneta/quick win styling

### PR DESCRIPTION
**Pull request by krezka**
and fix of "isEqualTo is not a function"

### CODE REVIEW

File: src/exchange/depthChart/depthchart.ts [hunk 1]
```diff
@@ -351,7 +351,7 @@ function sellsAfterForSellOffer(remaining: BigNumber, price: BigNumber, sells: O
   let remainingUnused = true;
   const sellsResult = sells.reduce(
     (result: Offer[], offer: Offer) => {
-      if (remainingUnused && price.isEqualTo(offer.price)) {
+      if (remainingUnused && price.eq(offer.price)) {
```
Comment (peculiarity) [inline src/exchange/depthChart/depthchart.ts]: Why did you change from `isEqualTo` to `eq`?

File: src/exchange/priceChart/PriceChartView.tsx [hunk 2]
```diff
@@ -332,8 +335,9 @@ function axes(_data: PriceChartDataPoint[],
   // style y labels
   yCandleAxisGroup.selectAll('.tick text')
     .classed(styles.axisYMainLabel, true);
-  yVolumeAxisGroup.selectAll('.tick text')
-    .classed(styles.axisYVolumeLabel, true);
+  // style volume labels
+  // yVolumeAxisGroup.selectAll('.tick text')
```
Comment (peculiarity) [inline src/exchange/priceChart/PriceChartView.tsx]: If you don't need it just remove it

File: src/exchange/depthChart/depthchart.ts (hunk 1 above)
Comment (krzkaczor) [inline src/exchange/depthChart/depthchart.ts]: what's the difference? According to docs its the same(?) https://mikemcl.github.io/bignumber.js/

Comment (krezka) [reply src/exchange/depthChart/depthchart.ts]: because isEqualTo does not exists in this BigDecimal
